lib/bloc/details_bloc.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:marvel/bloc/marvel_event.dart';
+import 'package:marvel/bloc/marvel_state.dart';
+import 'package:marvel/data/model/series.dart';
+import 'package:marvel/data/repository/marvel_repository.dart';
+
+class DetailsBloc extends Bloc<MarvelEvent, MarvelState> {
```

Comment:
For each bloc should be dedicated events and states like DetailsEvent and DetailsState